content/docs/getting-started/hello-world.md

```diff
@@ -38,6 +38,10 @@ use tokio::prelude::*;
 # fn main() {}
 ```
 
+Here we use Tokio's own [`io`] and [`net`] modules. These modules provide the same
+abstractions over networking and I/O-operations as the corresponding modules in `std`,
```

Comment:
nit: the `,` after `std` is not necessary

Reply:
Agreed :+1: